src/components/renders/filters/renderColors.ts

```diff
@@ -0,0 +1,19 @@
+import { Product } from '../../../types';
+
+export function renderColors(data: Array<Product>) {
+  const colors: Array<string> = [];
```

Comment:
Это не то, чтобы неправильно, как ты сделал сейчас, но я обычно предпочитаю на названия обычных типов обозначать массив через литерал, а не через Array. Его я использую, если внутрь что-то большое вкладывать, например объект (`Array<{name: string}>`). Из опыта это позволяет немного повысить читаемость.
```suggestion
  const colors: string[] = [];
```